recipes/pyzmq/build.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/bash
 
-export DYLD_LIBRARY_PATH="$PREFIX/lib:$DYLD_LIBRARY_PATH"
```

Comment:
We really need this line for Mac unfortunately. I was unable to build locally without it either.